src/client/routes/lobby/GameList.tsx

```diff
@@ -1,46 +1,40 @@
 import * as React from "react";
-import { withRouter } from "react-router-dom";
+import { withRouter, RouteComponentProps } from "react-router-dom";
 import List from "@material-ui/core/List";
 import ListItem from "@material-ui/core/ListItem";
-import Divider from "@material-ui/core/Divider";
+import ListItemText from "@material-ui/core/ListItemText";
 import ApiService from "../../common/ApiService";
 import { withStyles } from "@material-ui/core/styles";
+import { Paper, WithStyles, createStyles, Theme, Typography, Button } from "@material-ui/core";
+import AddIcon from "@material-ui/icons/Add";
```

Comment:
Unnecessasry deep path for module import

Reply:
You mean it should be like this? `import { AddIcon } from "@material-ui/icons";` Doing that would give error `has no exported member 'AddIcon'.`

Reply:
Reference the type "as is": 
import {Add} from "@material-ui/icons";
this should lead you through the @mui re-reference of SvgIcon as normal

Reply:
Using the import that way would import all icons from "@material-ui/icons" causing our bundle size rise around 8Mb.

We might want to use single separate imports for "@material-ui/core" as well as importing any component with "{ module } from "@material-ui/core" causes it to load 395K of code.

If we want to use those one line imports, we need to look into these options: https://material-ui.com/guides/minimizing-bundle-size/#option-2